Components/Card/card.html

```diff
@@ -0,0 +1,341 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../Styles/doc.css">
+    <link rel="stylesheet" href="./card.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <title>Card | Plume UI</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <h1>Plume UI</h1>
+        </div>
+        <div class="hamburger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+        </div>
+    </header>
+    <!-- Sidebar -->
+    <aside>
+        <nav>
+            <a href="../Usage/usage.html">Getting Started</a>
+            <a href="../Avatar/avatar.html">Avatar</a>
+            <a href="../Alert/alert.html">Alert</a>
+            <a href="../Badge/badge.html">Badge</a>
+            <a href="../Button/button.html">Button</a>
+            <a href="#" class="active">Card</a>
+            <a href="../Image/image.html">Image</a>
+            <a href="../Input/input.html">Input</a>
+            <a href="../List/list.html">List</a>
+            <a href="../Nav/nav.html">Navigation</a>
+            <a href="../Toast/toast.html">Toast</a>
+        </nav>
+    </aside>
+    <!-- Content -->
+    <main>
+        <div class="content">
+            <h2 class="content-title">Card</h2>
+            <p class="content-brief">
+                Cards are great for organizing and enhancing a website's user interface. They should be easy to scan for relevant and actionable information.
+            </p>
+            <hr>
+            <div class="content-feature">
+                <h3>Cards with Badges</h3>
```

Comment:
Here in this card instead of adding a circle as a badge, you can try adding a "new" tag as a badge. 
code looks good 👍